src/java.prefs/unix/classes/java/util/prefs/FileSystemPreferences.java

```diff
@@ -53,22 +53,21 @@ class FileSystemPreferences extends AbstractPreferences {
         loadPrefsLib();
     }
 
-    @SuppressWarnings({"removal", "restricted"})
+    @SuppressWarnings("restricted")
     private static void loadPrefsLib() {
-        PrivilegedAction<Void> load = () -> {
+//        PrivilegedAction<Void> load = () -> {
             System.loadLibrary("prefs");
-            return null;
-        };
-        AccessController.doPrivileged(load);
+//            return null;
+//        };
+//        AccessController.doPrivileged(load);
     }
```

Comment:
Looks like there is some clean up here to do of the commented out code...

Reply:
Whoops!